test/Sandpit/Exscientia/Trajectory/test_trajectory.py

```diff
@@ -0,0 +1,48 @@
+import BioSimSpace.Sandpit.Exscientia as BSS
+
+import pytest
+
+@pytest.fixture
+def traj_mdtraj(scope="session"):
+    """A trajectory object using the MDTraj backend."""
+    return BSS.Trajectory.Trajectory(
+            trajectory="test/Sandpit/Exscientia/input/trajectories/ala.trr",
+            topology="test/Sandpit/Exscientia/input/trajectories/ala.tpr")
+
```

Comment:
Do you mind add a frame to test `traj.getFrames(index)`? Thanks.

Reply:
Yes, can do. It's implicitly tested in the coordinates check, but I can make it more explicit.

Reply:
Just to check, do you mean the standalone `Trajectory.getFrame` function? This extracts a frame from the trajectory without loading the whole thing into memory, so is handy for getting specific frames from large files. Currently this only uses `mdtraj`. I can try to adapt it to use both backends, but I'm not sure if that's possible.

Reply:
Yes, it works with both backends, although it's a bit slower with `MDAnalysis`. Will try to re-use the existing functionality so that `getFrame` behaves the same way as the `getTrajectory` method, only extracting a single frame directly.

Reply:
Thanks. My user case is
```
traj = Trajectory(process=None, trajectory='gromacs.xtc', topology='gromacs.tpr')
system = traj.getFrames(0)
```

Reply:
This works for me with the current code. At least with the example input used for the tests.

Reply:
```python
In [15]: config = process.getConfig()

In [16]: config.append("nstxout-compressed = 10000")

In [17]: process.setConfig(config)

In [18]: process.start()
Out[18]: BioSimSpace.Process.Gromacs(<BioSimSpace.System: nMolecules=631>, <BioSimSpace.Protocol.Custom>, exe='/usr/local/gromacs/bin/gmx', name='gromacs', work_dir='/home/lester/Code/BioSimSpace/demo/tmp', seed=None)

In [19]: process.wait()

In [20]: traj = BSS.Trajectory.Trajectory(trajectory="tmp/gromacs.xtc", topology="tmp/gromacs.tpr")

In [21]: traj
Out[21]: <BioSimSpace.Trajectory: nFrames=11, backend='mdanalysis'>

In [22]: traj.getFrames(0)
Out[22]: [<BioSimSpace.System: nMolecules=1>]
```

Reply:
One issues with `getFrames` is that, depending on the format, we can lose the original molecules in the system, i.e. what is returned is just a single molecule. Process objects have a way of remapping things, or using alternative means to extract frames, e.g. `trjconv`. Here we are just wrapping around `MDTraj` and `MDAnalysis`, so are limited by what they can read/write.

Reply:
So if a process is being passed to the trajectory, would the property of the system in the process be retained? Or it will be lost anyway?

Reply:
Thanks for the explanation. I'm happy with this PR being merged. @msuruzhon asked me to review this and I guess I will test myself to see if it works for my purpose.